apps/admin-page/src/app/app.component.ts

```diff
@@ -7,4 +7,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'admin-page';
+  public sidebarCollapsed = false;
+  public username = 'username@example.com';
```

Comment:
The component variables are public by default, making the keyword redundant. Please remove it
```suggestion
   sidebarCollapsed = false;
   username = 'username@example.com';
```